packages/drivers/local-driver/src/auth.ts

```diff
@@ -49,9 +48,12 @@ export function generateToken(
 }
 
 export function generateUser(): IUser {
+	const userId = uuid();
+	const userName = userId.match(/^([\da-f]{8})-([\da-f]{4})/); // userName takes the first two segments of the userId.
```

Comment:
I think this makes the intention a bit clearer, can we update them all?
```suggestion
	const userName = userId.match(/^([\da-f]{8})-([\da-f]{4})/); // Just use the first two segments of the (fake) userId as a fake name
```